extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -533,6 +535,17 @@ export default compose( [
 	withSelect( ( select, { clientId } ) => ( {
 		postId: select( 'core/editor' ).getCurrentPostId(),
 		innerButtons: select( 'core/editor' ).getBlocksByClientId( clientId ),
+		isInPremiumContentBlock: () => {
```

Comment:
Maybe a block `context` of `isPremiumContentChild` would be an easier way of handling this? or is there an issue with block contexts not being available for some Jetpack supported gutenberg releases?

Reply:
No issue, other than I haven't properly used block contexts before. Let me figure that out and I'll switch this up. :D

Reply:
@glendaviesnz I've now switched this to use block context. Much easier! You will need to apply this wp-calypso PR: https://github.com/Automattic/wp-calypso/pull/47245